src/Avatar.ts

```diff
@@ -142,7 +143,12 @@ export function avatarUrlForRoom(room: Room, width: number, height: number, resi
     if (room.isSpaceRoom()) return null;
 
     // If the room is not a DM don't fallback to a member avatar
-    if (!DMRoomMap.shared().getUserIdForRoomId(room.roomId)) return null;
+    if (
+        !DMRoomMap.shared().getUserIdForRoomId(room.roomId)
+        && !(room instanceof LocalRoom)
```

Comment:
we seem to generally check the room ID prefix rather than the room's type - should we be using `instanceof` checks instead? 

Also, it might be worth making this check a utility function somewhere:

```typescript
export function isLocalRoom(room: Room): boolean {
    return /* ... */;
}
```